packages/node-cli/src/utils/AuthenticationUtils.js

```diff
@@ -160,5 +174,43 @@ async function authenticateCi(params, sdkPath, projectFolder, executionEnvironme
 		.withCommandParameters(authenticateCiExecutionContext.getParams())
 		.build();
 }
+/**
+ * 
+ * @param {String} authid 
+ * @param {String} sdkPath 
+ * @param {ExecutionEnvironmentContext} executionEnvironmentContext 
+ * @returns {SdkOperationResult}
+ */
+async function checkIfReauthzorizationIsNeeded(authid, sdkPath, executionEnvironmentContext) {
```

Comment:
Was it intentional to have the "z" in authzorization?

Reply:
It's a mistake. I'll fix the typo.